shipment_advice_bill_auto_complete/models/shipment_advice.py

```diff
@@ -27,17 +28,13 @@ class ShipmentAdvice(models.Model):
         copy=False,
     )
 
-    @api.depends("planned_move_ids.purchase_line_id.order_id.partner_id")
+    @api.depends("planned_move_ids.picking_id.partner_id", "state", "shipment_type")
     def _compute_supplier_ids(self):
         for shipment in self:
-            if shipment.shipment_type != "incoming":
-                shipment.supplier_ids = False
-            elif shipment.state == "done":
-                shipment.supplier_ids = (
-                    shipment.loaded_move_line_ids.picking_id.partner_id
-                )
-            else:
-                shipment.supplier_ids = shipment.planned_move_ids.picking_id.partner_id
+            supplier_ids = False
+            if shipment.shipment_type == "incoming":
+                supplier_ids = shipment.planned_move_ids.picking_id.partner_id
```

Comment:
On a in coming shipment.advice/picking there will be no loaded move lines.
This was just working because this part of code was not reached before, the compute was not executed again. Because the planned moves not change. Now with the new unplan of incoming not received pickings it reached this part of code because the planned moves changed after the action_done -> which sets the state.

Reply:
I don't know if it is needed, maybe it is good to align it. 
But i don't think we need to change it within this PR.